frontend/apps/reader/modules/readerfont.lua

```diff
@@ -404,6 +433,52 @@ These fonts will be used in this order:
 You can set a preferred fallback font with a long-press on a font name, and it will be used before these.
 If that font happens to be part of this list already, it will be used first.]]),
             table.concat(self.ui.document.fallback_fonts, "\n")),
+    })
+    table.insert(settings_table, {
+        text = _("Adjust fallback font sizes"),
+        checked_func = function()
+            return G_reader_settings:nilOrTrue("cre_adjusted_fallback_font_sizes")
+        end,
+        callback = function()
+            G_reader_settings:flipNilOrTrue("cre_adjusted_fallback_font_sizes")
+            self.ui.document:setAdjustedFallbackFontSizes(G_reader_settings:nilOrTrue("cre_adjusted_fallback_font_sizes"))
+            self.ui:handleEvent(Event:new("UpdatePos"))
+        end,
+        help_text = _([[
+Adjust the size of each fallback font so they all get the same x-height, and lowercase characters picked in them look similarly sized as those from the defaut font.
+This may help with greek words inside latin text (as latin fonts often do not have all the greek characters), but may make Chinese or Indic characters smaller when picked from fallback fonts.]]),
```

Comment:
It should still be Greek and Latin with an uppercase letter when referring to the alphabet.